ee/clickhouse/models/event.py

```diff
@@ -284,6 +285,14 @@ def get_event_count_month_to_date() -> int:
     return result
 
 
+def get_billable_event_count_last_30_days() -> int:
+    last_30_days = timezone.now().replace(hour=0, minute=0, second=0, microsecond=0)
+    last_30_days = last_30_days - timedelta(days=30)
+
+    result = sync_execute(BILLABLE_EVENTS_COUNT, {"date_from": last_30_days, "date_to": timezone.now()})[0][0]
+    return result
```

Comment:
Maybe worth wrapping in a try/except statement in case the query fails/times out, the system status endpoint still works?